packages/wallet-backend-update/src/account/service.ts

```diff
@@ -12,18 +12,21 @@ type CreateAccountArgs = {
   assetId: string
 }
 
-type FundAccountArgs = {
+type FundsArgs = {
   userId: string
   amount: number
   assetCode: string
 }
+type FundAccountArgs = FundsArgs
+type WithdrawFundsArgs = FundsArgs
```

Comment:
```suggestion
type FundAccountArgs = {
  userId: string
  amount: number
  assetCode: string
}
type WithdrawFundsArgs = FundAccountArgs
```

Reply:
Fund account means that I am funding the account, so that's why I wanted to have a general FUNDS account, that is being used when funding and withdrawing